### DISCUSSION TITLE
actions-metrics-server documentation/help

### DISCUSSION THREAD

**Discussion by neilharris1** [actions/actions-runner-controller#2893] [Questions]
Hello,

I've recently enabled actions-metrics-server via my helm chart:

The metrics I'm interested in are the following:

github_workflow_job_run_duration_seconds
github_workflow_job_queue_duration_seconds
github_workflow_job_failures_total

However, when I run workflows, I'm not seeing any of these metrics in prometheus. 
The only reason I know this metrics should exist is because I looked at the source code. Is there any plan to document what actions-metrics-server does, what metrics are available, and how to configure it to expose these metrics? Because at the moment I can't find anything useful regarding this in the github docs.

**Comment by awalther52** [comment]
Hello, I have a similar issue. Did you figure out how to get the metrics server working?
